python-doctest/user.py

```diff
@@ -0,0 +1,46 @@
+# class User:
```

Comment:
Does it make sense to add a comment to say what to do with this file? Or maybe we rename the `User` to `User_one`, `User_two` etc, so that when they run it they can see everything?